modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
It marks the currently-installed one with `***`, so I'm using `grep -A1` to get the line after that, but I agree that this is...a bit cumbersome.